pyanaconda/modules/common/structures/partitioning.py

```diff
@@ -343,3 +343,233 @@ def format_options(self) -> Str:
+
+        Set where the device will be mounted.
+        For example: '/', '/home', 'none'
+
+        :return: a path to a mount point or 'none'
+        """
+        return self._mount_point
+
+    @mount_point.setter
+    def mount_point(self, mount_point: Str):
+        self._mount_point = mount_point
+
+    @property
+    def format_type(self) -> Str:
+        """Format of the device.
+
+        For example: 'xfs'
+
+        :return: a specification of the format
+        """
+        return self._format_type
+
+    @format_type.setter
+    def format_type(self, format_type: Str):
+        self._format_type = format_type
+
+    @property
+    def label(self) -> Str:
+        """Label of the file system.
+
+        :return: a string
+        """
+        return self._label
+
+    @label.setter
+    def label(self, label: Str):
+        self._label = label
+
+    @property
+    def luks_version(self) -> Str:
+        """LUKS format version.
+
+         Supported values:
+            luks1
+            luks2
+
+        :return: a name of the version
+        """
+        return self._luks_version
+
+    @luks_version.setter
+    def luks_version(self, value: Str):
+        self._luks_version = value
+
+    @property
+    def device_name(self) -> Str:
+        """Name of the device.
+
+        :return: a string
+        """
+        return self._device_name
+
+    @device_name.setter
+    def device_name(self, value):
+        self._device_name = value
+
+    @property
+    def device_size(self) -> UInt64:
+        """Size of the device
+
+        :return: a size in bytes
+        """
+        return UInt64(self._device_size)
+
+    @device_size.setter
+    def device_size(self, size: UInt64):
+        self._device_size = size
+
+    @property
+    def device_raid_level(self) -> Str:
+        """RAID level of the device.
+
+        For example: raid0
+
+        :return: a string
+        """
+        return self._device_raid_level
+
+    @device_raid_level.setter
+    def device_raid_level(self, value):
+        self._device_raid_level = value
+
+    @property
+    def device_encrypted(self) -> Bool:
+        """Encrypt the device.
+
+        :return: True or False
+        """
+        return self._device_encrypted
+
+    @device_encrypted.setter
+    def device_encrypted(self, value):
+        self._device_encrypted = value
+
+    @property
+    def container_name(self) -> Str:
+        """Name of the container.
+
+        :return: a string
+        """
+        return self._container_name
+
+    @container_name.setter
+    def container_name(self, value):
+        self._container_name = value
+
+    @property
+    def container_size_policy(self) -> Int64:
+        """The container size policy.
+
+        Supported values:
+            -1  As large as possible
+             0  Automatic size
+            >0  Fixed size in bytes
+
+        :return: a size policy
+        """
+        return Int64(self._container_size_policy)
```

Comment:
It was originally implemented this way, but it was too difficult to work with two values since Blivet uses only one. I will keep it in my mind for the future improvements.